laceworksdk/http_session.py

```diff
@@ -50,7 +51,8 @@ def __init__(self, account, subaccount, api_key, api_secret):
         # Set the base parameters
         self._api_key = api_key
         self._api_secret = api_secret
-        self._base_url = f"https://{account}.{DEFAULT_BASE_DOMAIN}"
+        self._base_domain = base_domain or DEFAULT_BASE_DOMAIN
```

Comment:
Ah.  Good catch.  I initially had it like this bought thought setting default in method instantiation was cleaner.